en/docs/administer/changing-to-ibm-db2.md

```diff
@@ -90,180 +161,116 @@ Follow the steps below to change the type of the default datasource.
-                            ...
-                        </configuration>
-                    </definition>
-            </datasource>
-            ```
-
-#### Configuring new  datasources to manage registry or user management data
-
-Follow the steps below to configure new datasources to point to
-the new databases you create to manage registry and/or user management
-data separately.
-
-1.  Add a new datasourcewithsimilar configurations as the
-    [`             WSO2_CARBON_DB            `
-    datasource](#changing-the-default-datasource) above to
-    the \<
-    `            IS_HOME>/repository/conf/datasources/           `
-    `            master-datasources.xml           ` file. Change its
-    elements with your custom values. For instructions, see [Setting up
-    datasource
-    configurations.](#setting-up-datasource-configurations)
-2.  If you are setting up a separate database to store registry-related
-    data, update the following configurations in the \<
-    `             IS_HOME>/repository/conf/            `
-    `             registry.xml            ` `            ` file.
-
-    ``` xml
-        <dbConfig name="wso2registry">
-            <dataSource>jdbc/MY_DATASOURCE_NAME</dataSource>
-        </dbConfig>
-    ```
-
-3.  If you are setting up a separate database to store user management
-    data, update the following configurations in the \<
-    `             IS_HOME>/repository/conf/            `
-    `             user-mgt.xml            ` file.
-
-    ``` xml
-    <Configuration>
-        <Property name="dataSource">jdbc/MY_DATASOURCE_NAME</Property>
-    </Configuration>
-    ```
-
-### Creating database tables
-
-To create the database tables, connect to the database that you created
-earlier and run the following scripts in the DB2 Express-C command
-editor.
-
-1.  To create tables in the registry and user manager database (
-    `            WSO2CARBON_DB           ` ), use the below script:
-
-    ``` java
-    <IS_HOME>/dbscripts/db2.sql
-    ```
-
-2.  Restart the server.
-
-You can create database tables automatically **when starting the product for the first time** by using the `           -Dsetup          `
-parameter as follows:
 
--   For Windows:
-    `             <IS_HOME>/bin/wso2server.bat -Dsetup            `
+### Configure the connection pool to commit pending transactions on connection return  
+        
+  1.  Navigate to either one of the following locations based on your OS.
+        -   On Linux/Mac OS:
+            `                 <IS_HOME>/bin/wso2server.sh/                `
+        -   On Windows:
+            `                 <IS_HOME>\bin\wso2server.bat                `
+  2.  Add the following JVM option:
+
+       ``` java
+       -Dndatasource.disable.rollbackOnReturn=true \
+       ```
+
+  3.  Navigate to the
+        `               <IS_HOME>/repository/conf/deployment.toml              `
+        file.
+  4.  Disable the `               defaultAutoCommit              `
```

Comment:
Disable the `               defaultAutoCommit              ` property

Reply:
done